CHANGELOG.md

```diff
@@ -1,3 +1,99 @@
+# [MongooseIM 6.1.0](https://github.com/esl/MongooseIM/releases/tag/6.1.0) - 2023-05-10
+
+## Highlights
```

Comment:
I think the update of Universal Analytics to Google Analytics 4 deserves the inclusion to highlights, but I'm not 100% sure.

Reply:
Yeah, you're right. I wrote the highlights section before the merge of the PR with GA4 and forgot to update it.